### ISSUE TITLE
"6kbps" in file name of config isn't equal to reality?

### ISSUE THREAD

**Issue by forwiat** [zhenye234/xcodec#12]
hello @zhenye234 @shwj114514 , thanks for your contribution in xcodec!
I have doubt when I read the codes. in codec_16k_6kbps_v3_vqdp.yaml, we can calculate "bps" in this way: n_q=8, bins=1024, downsample_rate=320. bit_rate=8 * log(1024) * 50=4000bps=4kbps. It isn't equal to "6kbps".
is it right?

**Comment by forwiat** [comment]
another question is: "target_bandwidths" in main_launch_vqdp.py, variable "bw" maybe "0.5" or "1.5", which means function "get_num_quantizers_for_bandwidth" always return 1 in vq.py.

**Comment by forwiat** [comment]
I think 6kbps is calculated by target_bandwidths=6 maybe.

**Comment by zhenye234** [comment]
Yes, you are right, it is 4kbps, because my base code used 6kbps as default setting, so I don't change the config name.

**Comment by forwiat** [comment]
thanks for answer!
